index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Collision Summit 2023</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <header id="social-header">
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href=""><img src="images/social/twitter-16.png" alt="twitter"></a></li>
+          <li><a href=""><img src="images/social/facebook-16.png" alt="facebook"></a></li>
+          <li><a href="#logout">Logout</a></li>
+          <li><a href="#mypage"> My Page</a></li>
+          <li><a href="#Portuguese"> Portuguese</a></li> 
+        </ul>
+      </nav>
+    </header>
+    <header>
+      <div class="logo">
+        <a href="index.html"
+          ><img src="images/Logo/logo.avif" alt="logo"   
+        /></a>
+      </div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#Contact">News</a></li>
+        </ul>
+      </nav>
+      <button type="button" id="header-button">Collision Campaign</button>
+    </header>
+    <div class="popup-menu">
+      <div class="menu-button">
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+      </div>
+      <div>
+        <ul>
+          <li>
+            <a href="index.html">Home</a>
+          </li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <a href="#Contact">News</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <main class="main-content">
+      <section id="main">
+        <div>
+          <h1>"Hello! Tech World"</h1>
+          <h2>WEB SUMMIT 2023</h2>
+          <p id="introduction">
+            Although times may be uncertain, one thing
+            remains true. There is a simple power in people
+            coming together.
+          </p>
+          <h3>2023.07.24(MON) ~ 28(FRI)</h3>
+          <p>ENNERCARE CENTRE, TORONTO - CANADA.</p>
+        </div>
+      </section>
+      <section id="programs">
+        <h2>Main Program</h2>
+        <hr class="solid" />
+        <div id="all-programs">
+          <article class="program-cards">
+            <img src="images/programs/program1.png" alt="lecture" />
+            <h2>Lecture</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program2.png" alt="Exhibition" />
+            <h2>Exhibition</h2>
+            <p>
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design, the fonts, and also please implement the `hover effect` on the cards).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164411069-f160413d-f98d-4ec2-8329-2462a25bd496.png) | ![image](https://user-images.githubusercontent.com/10332499/164411260-b7ea746f-562e-40c7-b61e-3acbe6bc2827.png)

Reply:
resolved